datalake/backend/tests/test_processing_stack.py

```diff
@@ -0,0 +1,133 @@
+import json
+import logging
+import os
+import time
+
+from pytest import mark
+
+from .utils import S3Object, any_safe_file_path
+
+ENV = os.environ.get("DEPLOY_ENV", "dev")
+
+APPLICATION_TAG_KEY = "ApplicationName"
+APPLICATION_TAG_VAL = "geospatial-data-lake"
+
+ENVIRONMENT_TAG_KEY = "EnvironmentType"
+ENVIRONMENT_TAG_VAL = ENV
```

Comment:
This constant is just a copy of the other one, so one of them can be inlined.

Reply:
done in 57de24f1545fd01666d5eec613b82066d8244122